PhotoBooth/PhotoShootSetupCardView.swift

```diff
@@ -2,7 +2,7 @@ import UIKit
 
 class PhotoShootSetupCard: UIView {
 
-    private let viewModel: PhotoShootSetupCardViewModel
+    private let viewModel: SetupPhotoShootViewModeling
```

Comment:
if the class is PhotoShootSetUpCard, should the view model protocol be `PhotoShootSetUpCardViewModeling`?